mage_ai/server/server.py

```diff
@@ -342,6 +343,43 @@ def make_app(template_dir: str = None, update_routes: bool = False):
         (r'/version-control', MainPageHandler),
     ]
 
+    if ENABLE_PROMETHEUS or OTEL_EXPORTER_OTLP_ENDPOINT:
```

Comment:
Did you mean to include the check for `ENABLE_PROMETHEUS` here? Or should it just be checking for `OTEL_EXPORTER_OTLP_ENDPOINT`? If it's just `OTEL_EXPORTER_OTLP_ENDPOINT`,  can we combine the code in the 2 `if` blocks?

Reply:
They share some of the same initialization code